src/Tools/ExternalAccess/Razor/Shared/Microsoft.CodeAnalysis.ExternalAccess.Razor.Shared.projitems

```diff
@@ -8,94 +8,4 @@
   <PropertyGroup Label="Configuration">
     <Import_RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.Razor.Shared</Import_RootNamespace>
   </PropertyGroup>
```

Comment:
do we need this project at all anymore?  can it just be deleted?

Reply:
Oh no, it can be deleted. I didn't notice it wasn't when removing in vs

Reply:
i deleted it :)